apps/ui/src/views/SpaceUser.vue

```diff
@@ -70,6 +70,15 @@ const navigation = computed(() => [
   // { label: 'Delegators', route: 'space-user-delegators', count: delegatesCount.value },
 ]);
 
+const hasInvalidDelegations = computed(() => {
+  if (!props.space.delegations.length) return true;
+
+  return props.space.delegations.some(
+    delegation =>
+      !delegation.apiUrl || delegation.apiType === 'split-delegation'
+  );
+});
+
```

Comment:
If I understand correctly we assume that `split-delegation` would be the only delegation, so if one of delegations is split-delegation then we hide this button.

I'd still make it so we hide the button if **all** delegations are unsupported and in ModalDelegation only show supported delegations in the dropdown.

Reply:
Added.

- This space now has both valid and invalid delegations: http://localhost:8080/#/s:test.wa0x6e.eth/delegates
- Go to http://localhost:8080/#/s:test.wa0x6e.eth/profile/0x91FD2c8d24767db4Ece7069AA27832ffaf8590f3
- The Delegate modal should use the valid delegation
- If a space has 2 or more valid delegations, it should show the delegation schema selector, without any invalid delegations